Blueprints/BlueprintDefinitions/vs2017/ChatBotTutorial/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj

```diff
@@ -9,6 +9,6 @@
   <ItemGroup>
     <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
     <PackageReference Include="Amazon.Lambda.LexEvents" Version="3.0.0" />
-    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.0.0" />
+    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.0.1" />
```

Comment:
Fix blueprints - correct version for Amazon.Lambda.Serialization.Json is `2.1.0`

Reply:
Fixed.